gnucash/gnome/gnc-plugin-budget.c

```diff
@@ -126,6 +152,7 @@ gnc_plugin_budget_class_init (GncPluginBudgetClass *klass)
     plugin_class->actions      = gnc_plugin_actions;
     plugin_class->n_actions    = gnc_plugin_n_actions;
     plugin_class->ui_filename  = PLUGIN_UI_FILENAME;
+    plugin_class->add_to_window = add_to_window;
```

Comment:
I think you need to add...
   ` plugin_class->remove_from_window = remove_from_window;`

and then add
`static void remove_from_window (GncPlugin *plugin, GncMainWindow *window, GQuark type)`
`{`
   ` g_signal_handlers_disconnect_by_func (window, G_CALLBACK(page_changed), plugin);`
`}`